support/cas-server-support-cassandra-ticket-registry/src/main/java/org/apereo/cas/config/CassandraProperties.java

```diff
@@ -0,0 +1,80 @@
+package org.apereo.cas.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * This is {@link CassandraProperties}.
+ *
+ * @author David Rodriguez
+ * @since 5.1.0
+ */
+@ConfigurationProperties("cassandra")
+public class CassandraProperties {
+
+    private String contactPoints;
+    private String username;
+    private String password;
+    private String tgtTable;
+    private String stTable;
+    private String expiryTable;
+    private String lastRunTable;
+
+    public String getContactPoints() {
+        return contactPoints;
+    }
+
+    public void setContactPoints(final String contactPoints) {
+        this.contactPoints = contactPoints;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(final String username) {
+        this.username = username;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(final String password) {
+        this.password = password;
+    }
+
+    public String getTgtTable() {
+        return tgtTable;
+    }
+
+    public void setTgtTable(final String tgtTable) {
```

Comment:
I totally agree with you two but, the problem is that there is important configuration at the keyspace level like the replication.
Currently we are using CAS with cassandra in a single datacenter and a single keyspace but we are planning to go multi-datacenter. We would have 2 keyspaces, one for TGTs (and all the subtypes) that would be replicated to the others datacenters, and antoher keyspace for STs, not replicated.
That's why we implemented the ```getTgtTable()``` method, in order to make configurable the keyspace configuration as the table name would be ```keyspaceName.tableName```.

I guess for any storage system CAS supports the user could configure replication, backups, failover, ... How it's done?